kebechet/managers/update/update.py

```diff
@@ -922,6 +915,28 @@ def run(self, labels: list = []) -> Optional[dict]:
                             )
                         result = {}
                 results[self.runtime_environment] = result
-            close_manual_update_issue()
+
+            if overlays_dir:
+                issue = self.get_issue_by_title(_ISSUE_MANUAL_UPDATE)
+                if issue:
+                    for pr in self._pr_list:
+                        issue.comment(
+                            CLOSE_MANUAL_ISSUE_COMMENT_PR.format(
+                                pr=pr,
+                                time=datetime.now().strftime("%m/%d/%Y, %H:%M:%S"),
+                            ),
+                        )
+                    issue.close()
+
+            else:
+                close_manual_update_issue = partial(
```

Comment:
This doesn't need to be a partial as it is only called once. Creating partial functions is useful when you have a function getting called with many of the same parameters throughout a piece of code. It allows you to create an intermediate function that has certain parameters bound.

Reply:
ack